cmd/manager/main.go

```diff
@@ -97,7 +100,7 @@ func main() {
 		caCertDir                 string
 		globalPullSecret          string
 	)
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8443", "The address the metric endpoint binds to.")
```

Comment:
I would suggest mirroring `catalogd`'s options for certificates, rather than using fixed names below. This will provide flexibility (i.e. downstream).